app/views/pages/home.html.erb

```diff
@@ -1,10 +1,10 @@
 
-<div class="container d-flex flex-column justify-content-center mt-5 w-50 p-5" style="background-color:white">
-    <h1>Logo</h1><br>
+<div class="container d-flex flex-column justify-content-center my-5 w-50 py-4 px-5" style="background-color:white">
+    <%= image_tag ("profile.png"), alt: "chomp-logo", class: "w-100 my-3" %>
     <%= simple_form_for :chomp_session, url: root_path, method: :get do |f| %>
-            <div class="search-form-control form-group">
-            <input class="form-control string required" type="text" name="chomped" id="chomped" placeholder="Enter Session Code"/>
-            </div>
+        <div class="search-form-control form-group">
+            <input class="form-control string required" type="text" name="chomped" id="chomped" placeholder="Enter Session Code" style="text-align: center"/>
```

Comment:
instead of style="text-align: center", can you try class="text-center"? to avoid inline style